amazon_cloud_code_generator/cmd/refresh_examples.py

```diff
@@ -1,127 +1,89 @@
 #!/usr/bin/env python3
 
 import argparse
-import collections
 import io
 import pathlib
+from typing import Dict, List
 import ruamel.yaml
 import yaml
 
 
-def _task_to_string(task):
+def _task_to_string(task: Dict) -> str:
     a = io.StringIO()
     _yaml = ruamel.yaml.YAML()
```

Comment:
```suggestion
    _yaml = ruamel.yaml.YAML()
    _yaml.width = 160
```
It's not worth blocking the PR on this, but we may want to consider increasing the default width when dumping the yaml here. If you look at the examples for the eks_cluster module, you can see it has split some of the lines. The generated yaml is still valid, but it does look a little weird. I'm not sure what a good value for the width is.

Reply:
@gravesm Updated, thank you for that! It seems 160 also works for eks_fargate_profile. We can update it in the future if needed!